.github/workflows/stale-issue-bot.yaml

```diff
@@ -0,0 +1,15 @@
+name: "Close stale issues"
+on:
+  schedule:
+  - cron: "0 0 * * *"
+jobs:
+  stale:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/stale@v3
+      with:
+        repo-token: ${{ secrets.GITHUB_TOKEN }}
+        stale-issue-message: 'This issue has been marked as stale because it has been open for 90 days with no activity. This thread will be automatically closed in 30 days if no further activity occurs.'
+        exempt-issue-labels: 'keep open'
```

Comment:
Can we add the label for helm v4 issues to this?

Reply:
good call.

Reply:
I think I found an open issue with multiple labels using spaces via https://github.com/actions/stale/issues/98. I'll see if I can write a test case to see if that may be at play here as well, and if we need to submit a patch upstream before we can add the v4 label to the exemption list.

Reply:
https://github.com/actions/stale/issues/98#issuecomment-676649307